packages/http-client-java/generator/http-client-generator-test/src/main/java/com/cadl/longrunning/models/LroOperationStatusError.java

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.cadl.longrunning.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.core.models.ResponseError;
+import com.azure.core.util.CoreUtils;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+import java.time.OffsetDateTime;
+
+/**
+ * The LroOperationStatusError model.
+ */
+@Immutable
+public final class LroOperationStatusError implements JsonSerializable<LroOperationStatusError> {
```

Comment:
Caused by [case1](https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#1-responses-header-schema-has-public-access-in-tcgc) and [case2](https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#2-lro-initial-response-has-public-access-in-tcgc)